getRubric.js

```diff
@@ -2,64 +2,74 @@ const { flatten } = require('ramda')
 const canvasAPI = require('node-canvas-api')
 
 async function getRubric (courseId, assignmentId, rubricId) {
-  const [TAs, submissions, rubric, sections] = await Promise.all([
-    canvasAPI.getUsersInCourse(courseId, canvasAPI.getOptions.users.enrollmentType.ta),
-    canvasAPI.getAssignmentSubmissions(courseId, assignmentId, canvasAPI.getOptions.submissions.submission_comments),
+  const [enrollments, submissions, rubrics, sections] = await Promise.all([
+    canvasAPI.getEnrollmentsInCourse(courseId),
+    canvasAPI.getAssignmentSubmissions(courseId, assignmentId, canvasAPI.getOptions.submissions.submission_comments, canvasAPI.getOptions.submissions.rubric_assessment),
     canvasAPI.getRubric(courseId, rubricId),
-    canvasAPI.getSections(courseId, canvasAPI.getOptions.users.include.students)
-      .then(sections => flatten(
-        sections
-          .map(({ name, students }) => students
-            .map(student => ({ ...student, section: name }))
-          )
-      ))
+    canvasAPI.getSections(courseId)
```

Comment:
I noticed that you got rid of the:

```js
 .then(sections => flatten(
        sections
          .map(({ name, students }) => students
            .map(student => ({ ...student, section: name }))
          )
      ))
```
Just wanted to double check that this is intentional?

Reply:
Yes - we just need the section name